Library/Homebrew/os/mac/linkage_checker.rb

```diff
@@ -1,26 +1,52 @@
 require "set"
 require "keg"
 require "formula"
+require "os/mac/store/linkage_store"
 
 class LinkageChecker
-  attr_reader :keg, :formula
-  attr_reader :brewed_dylibs, :system_dylibs, :broken_dylibs, :variable_dylibs
-  attr_reader :undeclared_deps, :unnecessary_deps, :reverse_links
+  attr_reader :keg, :formula, :store
 
   def initialize(keg, formula = nil)
-    @keg = keg
+    @keg     = keg
```

Comment:
Just leave this formatting as-is; I'm not opposed to having it like this for new files but it makes it trickier to review when there's unrelated formatting changes.